ci/jobs/fast_test.py

```diff
@@ -55,6 +55,8 @@ def clone_submodules():
         "contrib/incbin",
         "contrib/yaml-cpp",
         "contrib/corrosion",
+        "contrib/limonp",
```

Comment:
The list defines which submodules are compiled into ClickHouse for the "Fasttests" CI check (which is supposed to be a smoke test). Both new submodules are not strictly necessary. Let's remove them from here.